File: core/src/com/alexpi/awesometanks/entities/items/HealthPack.java
package com.alexpi.awesometanks.entities.items;

import com.alexpi.awesometanks.utils.RandomUtils;

import com.badlogic.gdx.math.Vector2;

/**
 * Created by Alex on 19/02/2016.
 */
public class HealthPack extends Item {
    public int getHealth() {
        return health;
    }

    private final int health;
    public HealthPack( Vector2 position) {
        super("sprites/health_pack.png", position,.4f);
        health = RandomUtils.getRandomInt(100,200);
    }
}

File: core/src/com/alexpi/awesometanks/game/items/HealthPack.kt
package com.alexpi.awesometanks.game.items

import com.alexpi.awesometanks.game.utils.RandomUtils
import com.alexpi.awesometanks.screens.game.stage.GameContext
import com.badlogic.gdx.math.Vector2

/**
 * Created by Alex on 19/02/2016.
 */
class HealthPack(gameContext: GameContext, position: Vector2) : Item(gameContext,"sprites/health_pack.png", position, .4f) {
    val health: Int = RandomUtils.getRandomInt(100, 200)
}
